functionnal_tests/cypress/integration/activity_feed/workspace_activity_feed.spec.js

```diff
@@ -0,0 +1,99 @@
+import { PAGES } from '../../support/urls_commands'
+import { SELECTORS as s } from '../../support/generic_selector_commands'
+import defaultAdmin from '../../fixtures/defaultAdmin.json'
+import baseUser from '../../fixtures/baseUser.json'
+
+describe("The space activity feed page", () => {
+  const workspaceId = 1
```

Comment:
It is safer to use the values of the fixtures. This way you will be sure that the space has been created even if at some point someone changes the base fixture id.

Reply:
Done.